test/dummy/app/components/meta_header/meta_header.yml

```diff
@@ -0,0 +1,13 @@
+:title: 'My Meta Header'
+:meta: 'This describes the use of components as a whole'
+:stubs:
+  -
+    :mv_title: "Specific Example"
```

Comment:
I lean toward hiding any keys with mv_ in the properties section on the component information page.  Anyone think I should NOT hide them?